CMakeLists.txt

```diff
@@ -166,20 +185,15 @@ IF(MR_EMSCRIPTEN)
   ENDIF() # NOT MR_DISABLE_EMSCRIPTEN_ASYNCIFY
 ENDIF() # MR_EMSCRIPTEN
 
-set(IMGUI_DIR ${MESHLIB_THIRDPARTY_INCLUDE_DIR}/imgui)
-include_directories(${IMGUI_DIR})
-
-enable_testing()
-
 IF(NOT MR_EMSCRIPTEN)
 
-find_package(glfw3 CONFIG REQUIRED)
-pkg_check_modules(GTKMM gtkmm-3.0)
 pkg_check_modules(JSONCPP jsoncpp)
 include_directories(${JSONCPP_INCLUDE_DIRS})
 
-IF(${MR_FEDORA} OR ${MR_UBUNTU22})
- add_compile_definitions(SPDLOG_FMT_EXTERNAL)
+find_package(fmt)
```

Comment:
do we need it here?

Reply:
I think, yes
1. fmt has `fmt-config.cmake` ([link](https://ubuntu.pkgs.org/22.04/ubuntu-universe-amd64/libfmt-dev_8.1.1+ds1-2_amd64.deb.html)), so it's better to use their configuration
2. fmt is widely used in all of our sublibs